css/navbar.css

```diff
@@ -0,0 +1,109 @@
+.navbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  z-index: 2;
+  background-image: linear-gradient(
+    180deg,
+    #24211e 0%,
+    #5c544d8c 50%,
+    rgba(37, 33, 30, 0) 100%
+  );
+  transition: var(--transition-one);
+}
+
+.navbar * {
+  color: var(--clr-grey-10);
+}
+
+.navbar:hover {
+  background: var(--primary-color);
+}
+
+.nav {
+  padding: 0.625rem 1.5rem;
+  display: flex;
+}
+
+.information {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin-right: auto;
+  font-size: 12px;
+  z-index: 1;
+}
+
+.brand-logo {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  margin: 10px 0 0 0;
```

Comment:
no rem?

Reply:
Anything around 10px or less won't affect the responsiveness. Vikram made sense when he mentioned this.